tests/TestCase/Datasource/PaginatorTest.php

```diff
@@ -29,8 +29,12 @@ class PaginatorTest extends TestCase
      * @var array
      */
     protected $fixtures = [
-        'core.Posts', 'core.Articles', 'core.ArticlesTags',
-        'core.Authors', 'core.AuthorsTags', 'core.Tags',
+        'core.Posts',
+        'core.Authors',
+        'core.Articles',
+        'core.Tags',
+        'core.ArticlesTags',
+        'core.AuthorsTags',
```

Comment:
Did these need to be re-ordered to satisfy foreign key constraints?

Reply:
I was hacking through trying to get it to work with the existing code. I've thrown that away and just separated the queries.